x/bank/client/rest/sendtx.go

```diff
@@ -32,8 +34,12 @@ func init() {
 }
 
 // SendRequestHandlerFn - http request handler to send coins to a address.
-func SendRequestHandlerFn(cdc *codec.Codec, kb keys.Keybase, cliCtx context.CLIContext) http.HandlerFunc {
+func SendRequestHandlerFn(cdc *codec.Codec, kbInit func() (keys.Keybase, error), cliCtx context.CLIContext) http.HandlerFunc {
 	return func(w http.ResponseWriter, r *http.Request) {
+		kb, err := kbInit()
```

Comment:
Should we close the keybase (close the underyling DB) when we're done handling the request?